backend/api/routes/users.py

```
@@ -0,0 +1,785 @@
+        logger.warning(f"Invalid Stripe webhook signature: {message}")
+        raise HTTPException(status_code=400, detail=message)
+
+    # Handle the event
+    event_type = event.get("type")
+    logger.info(f"Received Stripe webhook: {event_type}")
+
+    if event_type == "checkout.session.completed":
+        session = event["data"]["object"]
+
+        # Process the completed checkout
+        success, user_email, credits, msg = stripe_service.handle_checkout_completed(session)
```

Ruff (0.14.10)

Unpacked variable `msg` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)